README.md

```diff
@@ -84,6 +84,13 @@ $ rails s
 - Twitter: [@DGuishny](https://twitter.com/DGuishny)
 - LinkedIn: [Guishny Desor](https://www.linkedin.com/in/guishny-desor-5421a01a9/)
 
```

Comment:
- [ ] OPTIONAL
Kindly deploys the application using Heroku, Netlify or GitHub Pages, and ensure the link to the deployed project is present in the README.